interface/forms/procedure_order/new.php

```diff
@@ -669,6 +678,40 @@ class="deleter"
                   format: "<?= $DateFormat; ?>"
               });
               $.datetimepicker.setLocale('<?= $DateLocale;?>');
+
+              var paper = <?php echo json_encode($paper);?>;
+              $('#procedure_type_names').on('blur', function(){
```

Comment:
Tell me what you are doing here? with this case statement.

Reply:
Uh, gives you a print button if the index==1 in case "imaging"?

Reply:
@teryhill I extracted the data from the database all at once for paper being selected and linked it with selected procedure using case statement with the help of index number.I used the json_encode function to transfer PHP varaible values to JavaScript.

Reply:
@aethelwulffe are we not supposed to have Print button for paper even if paper is selected??

Reply:
@2208Abhinav  You will need to read the variables. This is a list that can be changed by the user. It can be added to or have fields removed and names can be changed. Hard coding them will be a problem.

Reply:
@teryhill I understand that you are saying not to depend on **index**, directly extract the value from the variable because the list can be changed and the code that I have written is not dynamic when its using the case statement to provide index.

Reply:
I will have it solved by tomorrow positively :smiley:

Reply:
@2208Abhinav  You are doing great. I like your thought process.

Reply:
@2208Abhinav  are you able to test this on your system? Do you need some lab data ?

Reply:
@teryhill I will be glad if you provide the lab data for exact interpretation, right now I am working for the **index** problem.